src/libs/elektra/keyname.c

```diff
@@ -1580,23 +1646,24 @@ ssize_t keyAddBaseName (Key * key, const char * baseName)
  * (dot-dot), @c "%" (empty basename)). They will be properly escaped
  * and will not have their usual meaning.
  *
- * @see keyname for more details on special names
- *
- * If you want to add and not change the basename, use keyAddBaseName()
- * instead. If you do not want escaping, use keyAddName() instead.
+ * If you want to add to the basename instead of changing it, use keyAddBaseName().
+ * If you do not want any escaping, use keyAddName().
  *
- * @see keyAddBaseName() to add a basename instead of changing it
- * @see keyAddName() to add a name without escaping
- * @see keySetName() to set a completely new name
+ * @param key the Key whose basename to set
+ * @param baseName the new basename for the Key
  *
- *
- * @param key the key object to work with
- * @param baseName the string used to overwrite the basename of the key
  * @return the size in bytes of the new key name
- * @retval -1 on NULL pointers in key
- * @retval -1 if key was inserted to a keyset before
+ * @retval -1 on NULL pointers
+ * @retval -1 if Key was inserted to a KeySet before
```

Comment:
Maybe
```suggestion
 * @retval -1 if Key was inserted into KeySet before
```

Reply:
fixed in https://github.com/ElektraInitiative/libelektra/pull/3795/commits/cea2c469decb39e99b3d71197f27ed40dede38d9